src/pages/OneResourcePage/OneResourcePage.js

```diff
@@ -224,35 +225,19 @@ const OneResourcePage = ({
 }) => {
   const theme = useTheme()
   const resourceId = match.params.id
-  const [initiative, setInitiative] = useState(null)
-  const [isLoading, setIsLoading] = useState(true)
+  const [initiatives, isLoading] = useFetchedInitiatives()
   const [votesCount, setVotesCount] = useState(null)
   const [alreadyUpvoted, setAlreadyUpvoted] = useState(
     isAlreadyUpvoted(resourceId)
   )
 
-  useEffect(() => {
-    const fetchInitiatives = async () => {
-      await axios
-        .get(getJsonLink('resources.json'),
-          { headers: {'Access-Control-Allow-Origin': origin} })
-        .then(res => {
-          const { data } = res.data
-          const singleInitiative =
-            resourceId
-              ? find(data, ['id', resourceId])
-              : null
-          setInitiative(singleInitiative)
-        })
-        .catch(err => {
-          console.error('Nie udało się pobrać inicjatyw: ', err.message)
-        })
-        .finally(() => {
-          setIsLoading(false)
-        })
-    }
-    fetchInitiatives()
-  }, [resourceId])
+  const initiative = () => {
+    const singleInitiative =
+      resourceId
+        ? find(initiatives, ['id', resourceId])
+        : null
+    return singleInitiative
+  }
```

Comment:
do you think the below would work as expected ?
```suggestion 
const initiative = () =>
  resourceId ? find(initiatives, ['id', resourceId]) : null
```